opencti-platform/opencti-front/lang/front/en.json

```
@@ -3731,6 +3732,7 @@
   "You can configure your platform to consume OpenCTI Streams. A list of public and commercial native feeds is available in the": "You can configure your platform to consume OpenCTI Streams. A list of public and commercial native feeds is available in the",
   "You can enable/disable default values for marking in the customization of each entity type.": "You can enable/disable default values for marking in the customization of each entity type.",
   "You can now set a new password for your account.": "You can now set a new password for your account.",
+  "You cannot create a user for this feed since no group by default has been defined. ": "You cannot create a user for this feed since no group by default has been defined. ",
```

```suggestion
  "User cannot be created automatically for this feed since no default group has been defined. ": "User cannot be created automatically for this feed since no default group has been defined. ",
```

imo seems to be better like this